tsconfig.json

```
@@ -0,0 +1,35 @@
+{
+    "compilerOptions": {
+        "lib": [
+            "ES6",
+            "DOM"
+        ],
+        "baseUrl": ".",
+        "paths": {
+            "*": [
+                "src/*",
+            ]
+        },
+        "module": "ES2020",
+        "target": "ES5",
+        "strict": true,
+        "alwaysStrict": true,
+        "esModuleInterop": true,
+        "forceConsistentCasingInFileNames": true,
+        "noImplicitAny": true,
+        "jsx": "react",
+        "skipLibCheck": true,
+        "outDir": "./dist/",
+        "sourceMap": true,
+        "removeComments": true,
+        "moduleResolution": "node"
+    },
+    "include": [
+        "src/**/*.ts",
+        "src/**/*.tsx",
```

```suggestion
        "src/**/*.tsx",
        "test/**/*.ts",
```
ESLint complains that the test modules aren't included. This fixes it.